src/js/video.js

```diff
@@ -162,6 +162,11 @@ function videojs(id, options, ready) {
 
   options = options || {};
 
+  // Store a copy of the el before modification, if it is to be restored in destroy()
+  if (options.restoreEl === true) {
+    options.restoreEl = el.cloneNode(true);
```

Comment:
Not a block: Clever to re-use the option to store the clone element, though, that might be a bit unexpected.